eng/common/TestResources/Install-Modules.ps1

```diff
@@ -0,0 +1,16 @@
+$ModulesUrl = "https://pakrym0test0storage.blob.core.windows.net/myblobcontainer/AllModules.zip"
+$TempPath = Join-Path $([System.IO.Path]::GetTempPath()) $(New-Guid)
+$ZipFile = Join-Path $TempPath "Modules.zip"
+
+New-Item $TempPath -ItemType Directory -Force
+
+Invoke-WebRequest -Uri $ModulesUrl -OutFile $ZipFile 
+Expand-Archive $ZipFile -DestinationPath $TempPath -Force
+
+$p = [Environment]::GetEnvironmentVariable("PSModulePath")
```

Comment:
Instead of mucking around with appending to ```PSModulePath``` why don't we just do an ```Import-Module``` at the beginning of the PowerShell script.

Reply:
Because we need these modules for scripts that execute later in the pipeline.